lineage/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ plugins {
 dependencies {
   implementation(project(":common"))
   implementation(project(":core"))
+  implementation(project(":common"))
```

Comment:
Why do we write this line again, as it already appears in line 25.

Reply:
it's a mistake, removed